ui/docs/manual/using/integration/frontend.md

```diff
@@ -9,52 +9,32 @@ order: 10
-hybrid ("[regular web
-application](https://auth0.com/docs/clients#client-types)").
-
-This process supports any authentication and authorization your application
-needs for itself - displaying the user's name, storing user settings, or
-controlling access to resources based on the user identity.
-
-With a few extra parameters to the login process, this process will produce an
-access token which can also be exchanged for Taskcluster credentials as they
-are needed, by making an API call to the [login
-service](/docs/reference/integrations/taskcluster-login).
-
-The Taskcluster credentials have a very short expiration, but can be requested
+Taskcluster implements the [OAuth2 protocol](https://tools.ietf.org/html/rfc6749),
+supporting both the "Implicit" and "Authorization Code" flows.
+The "Resource Owner Password Credentials" and "Client Credentials" flows are not supported.
+
+Clients are pre-defined, and each pre-defined client indicates which flow it uses (and cannot use both).
+Some clients are whitelisted, meaning that user consent is not required.
+
+The Taskcluster credentials will eventually expire, but can be requested
 again when required. Callers should check the expiration before every call to a
 Taskcluster API and refresh when necessary.
 
 ## Creating a simple login integration
 
-Authorizing your application to use Taskcluster APIs from a frontend project is relatively
-straightforward, but there can be some hurdles to jump through. First, let's talk about a
-few prerequisites:
-
-* Possess an [auth0 Client](https://auth0.com/docs/clients) which can access Taskcluster
-credentials. If you don't have one, you can request a [New Single Sign On Application in
-The Hub](https://mozilla.service-now.com/sp?id=sc_cat_item&sys_id=1e9746c20f76aa0087591d2be1050ecb).
-Since this is going to be used on the frontend, make sure to request a client that uses
-"RS256 algorithms" and "OpenIDConnect (OIDC)". To be clear, you should also mention in the request
-comments that this is a "SPA (single-page application)", and that it "needs to use RS256 algorithms".
-* For ease and simplicity, use the [auth0.js client library](https://auth0.com/docs/libraries/auth0js/v8)
-and the [taskcluster-client-web](https://github.com/taskcluster/taskcluster-client-web) library.
-
-Now we can begin working on a simple login page. For brevity and demonstration purposes, we will
+Let’s begin working on a simple login page. For brevity and demonstration purposes, we will
 be using a single HTML file for this, but you can adapt these techniques into any more complex
-application, including those using ES imports. Here is the base page markup we are working with:
+application, including those using ES imports. We will also pretend the root URL of the
+Taskcluster deployment and the HTML demo file are hosted on
+`http://localhost:5080` and `http://localhost:4000` respectively. Here is the base page markup
```

Comment:
It might be easier for the reader to distinguish these, and read the first of them as a rootUrl, if it's `https://tc.example.com`.